v4/newrelic/datastore.go

```diff
@@ -9,26 +9,26 @@ type DatastoreProduct string
 
 // Datastore names used across New Relic agents:
 const (
-	DatastoreCassandra     DatastoreProduct = "Cassandra"
-	DatastoreCouchDB       DatastoreProduct = "CouchDB"
-	DatastoreDerby         DatastoreProduct = "Derby"
-	DatastoreDynamoDB      DatastoreProduct = "DynamoDB"
-	DatastoreElasticsearch DatastoreProduct = "Elasticsearch"
-	DatastoreFirebird      DatastoreProduct = "Firebird"
-	DatastoreIBMDB2        DatastoreProduct = "IBMDB2"
-	DatastoreInformix      DatastoreProduct = "Informix"
-	DatastoreMemcached     DatastoreProduct = "Memcached"
-	DatastoreMongoDB       DatastoreProduct = "MongoDB"
-	DatastoreMSSQL         DatastoreProduct = "MSSQL"
-	DatastoreMySQL         DatastoreProduct = "MySQL"
-	DatastoreNeptune       DatastoreProduct = "Neptune"
-	DatastoreOracle        DatastoreProduct = "Oracle"
-	DatastorePostgres      DatastoreProduct = "Postgres"
-	DatastoreRedis         DatastoreProduct = "Redis"
-	DatastoreRiak          DatastoreProduct = "Riak"
-	DatastoreSnowflake     DatastoreProduct = "Snowflake"
-	DatastoreSolr          DatastoreProduct = "Solr"
-	DatastoreSQLite        DatastoreProduct = "SQLite"
-	DatastoreTarantool     DatastoreProduct = "Tarantool"
-	DatastoreVoltDB        DatastoreProduct = "VoltDB"
+	DatastoreCassandra     DatastoreProduct = "cassandra"
+	DatastoreCouchDB       DatastoreProduct = "couchdb"
+	DatastoreDerby         DatastoreProduct = "derby"
+	DatastoreDynamoDB      DatastoreProduct = "dynamodb"
+	DatastoreElasticsearch DatastoreProduct = "elasticsearch"
+	DatastoreFirebird      DatastoreProduct = "firebird"
+	DatastoreIBMDB2        DatastoreProduct = "db2"
+	DatastoreInformix      DatastoreProduct = "informix"
+	DatastoreMSSQL         DatastoreProduct = "mssql"
+	DatastoreMemcached     DatastoreProduct = "memcached"
+	DatastoreMongoDB       DatastoreProduct = "mongodb"
+	DatastoreMySQL         DatastoreProduct = "mysql"
+	DatastoreNeptune       DatastoreProduct = "neptune"
+	DatastoreOracle        DatastoreProduct = "oracle"
+	DatastorePostgres      DatastoreProduct = "postgresql"
+	DatastoreRedis         DatastoreProduct = "redis"
+	DatastoreRiak          DatastoreProduct = "riak"
+	DatastoreSQLite        DatastoreProduct = "sqlite"
+	DatastoreSnowflake     DatastoreProduct = "snowflake"
+	DatastoreSolr          DatastoreProduct = "solr"
+	DatastoreTarantool     DatastoreProduct = "tarantool"
+	DatastoreVoltDB        DatastoreProduct = "voltDB"
```

Comment:
The otel spec is very clear about the names for each of these: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem